packages/components/menu/src/sub-menu.ts

```diff
@@ -51,6 +57,22 @@ export const subMenuProps = buildProps({
     type: Number,
     default: 6,
   },
+  expandCloseIcon: {
+    type: iconPropType,
+    default: '',
+  },
```

Comment:
How about changing the type here to this?

```ts
expandCloseIcon: {
	type: definePropType<Component | (() => VNode)>([Object, Function]),
},
```

Same in other places.

Reply:
Done

Reply:
Nice!